test/jdk/java/awt/Focus/FocusTraversalPolicy/ButtonGroupLayoutTraversal/ButtonGroupLayoutTraversalTest.java

```diff
@@ -50,98 +51,113 @@
 
 public class ButtonGroupLayoutTraversalTest {
 
-    static int nx = 3;
-    static int ny = 3;
+    private static final int nx = 3;
```

Comment:
Normally, as per coding standard, for static final modifier, the variable should be named in CAPS...but is there any need of making it final?

Reply:
Ran test 20 times on all OS and it passed, posted link in JBS. I avoided changing to CAPS to prevent code change noise and final is done since these variable values don't change or are constant throughout the code and we want it that way.

Reply:
I still think nx, ny should be made CAPS...It seems to be the case for static final constant variables in java/awt test folder...I don't think it will increase noise as it will impact only in l57

Reply:
It will affect more lines; these constants are used in for-loops to move focus between buttons as well as to validate the results. Let it be.

Reply:
Updated nx, ny with CAPS version.